### PR TITLE
[SNC-177] get element name

### PR DESCRIPTION
**Pull request by davidmdm**
Solutions Engineers desire a helper to get the name of elements as noted in this [issue](https://github.com/CircleCI-Public/circle-policy-agent/issues/35)

It is a common pattern in CircleCI config.yml to have elements by either a string or an object of a single key. We want to be able to return either the string value or the single key.

### CODE REVIEW

File: cpa/tester/policies/results.xml [hunk 1]
```diff
@@ -1,4 +1,15 @@
-<testsuites name="root" tests="38" failures="0" errors="0" time="0">
+<testsuites name="root" tests="46" failures="0" errors="0" time="0">
```
Comment (eddiewebb) [inline cpa/tester/policies/results.xml:1]: time will be populated in real output? (ELapsedMS in json today)

Comment (eddiewebb) [reply cpa/tester/policies/results.xml:1]: NM! I Saw this is in the Result.go.  Sweet!

Comment (davidmdm) [reply cpa/tester/policies/results.xml:1]: You know it! But when testing against static data, time is the enemy 🙈

**Review by rooneyshuman — APPROVED** [review]

File: cpa/tester/policies/helpers/utils/utils.rego [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org
+
+import data.circleci.utils
+
+policy_name["utils_tests"]
+
+test_to_array_scalar = utils.to_array(1) == [1]
+test_to_array_array = utils.to_array([1]) == [1]
```
Comment (sagar-connect) [inline cpa/tester/policies/helpers/utils/utils.rego:8]: also test_to_array_set?

**Review by sagar-connect — APPROVED** [review]

**Pull request merged**